### DISCUSSION TITLE
„People I follow“ as a Privacy Setting

### DISCUSSION THREAD

**Discussion by sirlupusdev** [Traewelling/traewelling#2554] [Ideas / Feature Requests]
It‘d be nice to have „Only People I follow“ as a privacy setting for trips to allow a more granular control over who can see trips without changing the complete profile to private. When only taking public profiles into account, there doesn‘t seem to be an „effective“ way to restrict the visibility of posts beyond Public/Private as of now when viewing the profile. 
The implementation may be similar to the „My Followers“ settings, although I didn‘t have a thorough look at the code. 
The general topic of this discussion is related to #2508.

**Comment by MrKrisKrisu** [comment]
Since this request was created, the "Trusted Users" feature has been introduced. I'm assuming that's sufficient, so I'll close this thread. :) I'd rather not add any more options, because otherwise it would just end up becoming too complex.
